src/eureka/S3_data_reduction/s3_reduce.py

```diff
@@ -271,6 +271,14 @@ def reduce(eventlabel, ecf_path=None, s2_meta=None):
                                                    data.mask.values,
                                                    log, name='V0')
 
+                # Start masking pixels based on DQ flags
+                # https://jwst-pipeline.readthedocs.io/en/latest/jwst/references_general/references_general.html
+                # Odd numbers in DQ array are bad pixels. Do not use.
```

Comment:
I can't see anything on that website that says only odd numbers are bad - where did you see this?

Reply:
Ah, gotcha